include/deal.II/base/symmetric_tensor.h

```diff
@@ -2039,6 +2064,49 @@ SymmetricTensor<rank_,dim,Number>::operator [] (const TableIndices<rank_> &indic
 
 
 
+template <int rank_, int dim, typename Number>
+inline
+Number *
+SymmetricTensor<rank_,dim,Number>::begin_raw()
+{
+  return &this->access_raw_entry(0);
```

Comment:
here and below: Is this a place where we ought to be using `std::addressof`?
http://en.cppreference.com/w/cpp/memory/addressof